documentation/firmware-management/index.rst

```diff
@@ -87,32 +86,29 @@ Upon discovery the Firmware Version property becomes read-only to prevent human
 ============================================
 Who can access Firmware Management features?
 ============================================
-As of Hyperview 3.4, any user can access Firmware Management features as long as:
-
-* You are using a Firmware Management-licensed instance (see :ref:`View license<View-license-doc>`.
-* You have access to at least one asset with managed firmware.
+As of Hyperview 3.4, any user can access Firmware Management features as long as you are using a Firmware Management-licensed instance (see :ref:`View license<View-license-doc>`.
 
 ====================================
 Navigating Firmware Management pages
 ====================================
 Hyperview features two main Firmware Management pages that can be considered "landing pages" for the feature set. These are:
 
-* The Firmware page (*Assets → Firmware*), which opens a grid of managed firmware that is grouped by manufacturer, asset type, and installed firmware version. It features a Details button at the row level to look up assets that have the same firmware version installed.
+* The Firmware page (*Assets → Firmware*), which opens a grid of managed and unmanaged firmware that is grouped by manufacturer, asset type, and installed firmware version. It features a Details button at the row level to look up assets that have the same firmware version installed.
 * The Information → Firmware page (asset → *Information → Firmware*), which opens a grid of installed and available firmware versions, as well as relevant action buttons.
```

Comment:
Use consistent capitalizations. "Asset" and "Firmware Versions" should be capitalized based on previous content.